main.go

```diff
@@ -112,6 +112,11 @@ func filterVariants(module, variant string, variantsMap gradle.Variants) (gradle
 func main() {
 	var config Configs
 
+	if debug := os.Getenv("is_debug"); debug == "yes" {
```

Comment:
this should be a field in Configs and an extra step input: Debug Mode under the Debug category in the step.yml

pls se an example here: https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/blob/master/step.yml#L165 and here: https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/blob/master/main.go#L38